tartiflette/sdl/query_resolver.py

```diff
@@ -0,0 +1,12 @@
+from tartiflette.sdl.schema import DefaultGraphQLSchema
+
+
+class QueryResolver(object):
```

Comment:
Missing a "how to use" documentation at the Class level.

Reply:
Added.